Source/Core/Core/NetPlayQoS.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <enet/enet.h>
+
+class NetPlayQoSSession
+{
+public:
+  explicit NetPlayQoSSession(ENetPeer* peer);
+  ~NetPlayQoSSession();
+
```

Comment:
You should delete the copy constructor and copy assignment operator for this class, so that someone won't accidentally create a copy and then get dangling pointers in the copy by having the original instance go out of scope.

Reply:
Fixed.